src/main/ts/Part2Ex2ArrayFunctions.ts

```diff
@@ -41,6 +41,7 @@ export const myFrogs: Frog[] = [
 
 export const runEach2 = (): void => {
   // TODO: Use Arr.each and console.log to print the name of each frog
+  Arr.each(myFrogs, (f: Frog) => console.log(f.name))
```

Comment:
```suggestion
  Arr.each(myFrogs, (f) => console.log(f.name))
```
You don't need to declare the type here as it'll be inferred. We normally try to avoid re-declaring redundant types as if you have to change it later it just makes more work 🤷